AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Notifications/ItineraryTeamLeadAssignedHandlerShould.cs

```diff
@@ -0,0 +1,57 @@
+using System.Text;
+using System.Threading.Tasks;
+using AllReady.Areas.Admin.Features.Notifications;
+using AllReady.Features.Notifications;
+using AllReady.Models;
+using MediatR;
+using Moq;
+using Xunit;
+
+namespace AllReady.UnitTest.Areas.Admin.Features.Notifications
+{
+    public class ItineraryTeamLeadAssignedHandlerShould
+    {
+        private const string itineraryName = "The itinerary";
+        private const string assigneeEmail = "test@test.com";
+        private const string assigneePhone = "+1 (999) 999-9999";
+        private const string itineraryUrl = "https://example.com";
+        private const string subject = @"You have been assigned as the Team Lead on an itinerary";
+        private string smsMessage = $@"You have been assigned as the Team Lead on the following itenerary: ""{itineraryName}"".";
+
+        [Fact]
+        public async Task SendTeamLeadAssignmentMessageToVolunteer()
+        {
+            var mockMediator = new Mock<IMediator>();
+            var handler = new IterneraryTeamLeadAssignedHandler(mockMediator.Object);
+
+            await handler.Handle(new IteneraryTeamLeadAssigned()
+            {
+                AssigneePhone = assigneePhone,
+                AssigneeEmail = assigneeEmail,
+                ItineraryName = itineraryName,
+                ItineraryUrl = itineraryUrl
+            });
+
+            var plaintextMsg = new StringBuilder();
+            plaintextMsg.AppendLine($@"You have been assigned as the Team Lead on the following itenerary: ""{itineraryName}"".");
```

Comment:
I think that this satisfies the goal of the test and ensures the functionality works, but we have to remember that this kind of test (with string literals) is likely fragile. I'm going to go ahead with the merge here, though, because ultimately we are going to move to sendgrid templates, so we'll be checking for a well-known template ID and not a series of strings.

Reply:
James, thanks for the feedback.  The test is problematic, I know (I experienced the fragility:)). Should also be broken into several, I think.  I'm willing to work on improving this with sendgrid templates once they're available.